docs/content/fsdocs-default.css

```diff
@@ -1,13 +1,11 @@
-/* @import url(https://fonts.googleapis.com/css?family=Droid+Sans|Droid+Sans+Mono|Open+Sans:400,600,700); */
-/* @import url('https://fonts.googleapis.com/css2?family=Hind+Vadodara&display=swap'); */
+/* CSS file from the fsprojects/FSharp.Formatting project modified by Atilim Gunes Baydin <gunes@robots.ox.ac.uk> for the DiffSharp project documentation, July 2021 */
```

Comment:
Because these are a small subset of contributions back to fsdocs I removed this if that's ok, will metion in the release notes

Reply:
That's fine